packages/config/src/index.ts

```diff
@@ -190,6 +190,8 @@ class TruffleConfig {
     // The require-nocache module used to do this for us, but
     // it doesn't bundle very well. So we've pulled it out ourselves.
     //@ts-ignore
+    //this doesn't do anything in the webpack build
+    //BUT may have an impact in non-webpack scenarios, ie, unit tests
```

Comment:
@gnidan, @davidmurdoch CI passing supports removing this cache/clean code. Still have to test develop/console behavior for contra cases.